composer.json

```diff
@@ -70,7 +70,12 @@
             "ZFTest\\Apigility\\Doctrine\\": "test/src/"
         }
     },
+    "bin": [
+        "./scripts/mongodb-shim"
```

Comment:
I was thinking about it, and I think we shouldn't have it here. Maybe it should be a part of https://github.com/doctrine/mongodb because of it we need to install that extra dependency on PHP 7+.

What I will do here... As we still support PHP 5.6 I would change default version in docker container to PHP 5.6 so then we don't need to install anything extra, and then describe in the docs (any maybe show message on container building) that for PHP 7+ we need install extra dependency. What do you think, @TomHAnderson ?

Reply:
1) I think your proposed solution is workable.
2) Using the least common denominator of PHP 5.6 is like [I don’t try to predict the future. I try to prevent it.](https://quoteinvestigator.com/2010/10/19/prevent-the-future/)  I could very well be overly pessimistic here.  I liked the pro-active approach better.  What about not installing the script in composer / 'everyone gets a copy' and instead installing in a bin/ directory local to the project so it is only ran from composer events?